commands.go

```diff
@@ -287,3 +288,23 @@ func unbanCommand(u *user, args []string) {
 	u.system("No user found!")
 
 }
+
+func helpCommand(u *user, _ []string) {
+	u.system("Welcome to Devzat! Devzat is chat over SSH: github.com/quackduck/devzat")
+	u.system("Because there's SSH apps on all platforms, even on mobile, you can join from anywhere.")
+	u.system("")
+	u.system("Interesting features:")
+	u.system("• Many, many commands. Run ./commands.")
+	u.system("• Rooms! Run ./room to see all rooms and use ./room #foo to join a new room.")
+	u.system("• Markdown support! Tables, headers, italics and everything. Just use \n in place of newlines.")
+	u.system("• Code syntax highlighting. Use Markdown fences to send code. Run ./example-code to see an example.")
+	u.system("• Direct messages! Send a quick DM using =user <msg> or stay in DMs by running ./room @user.")
+	u.system("• Timezone support, use ./tz Continent/City to set your timezone.")
+	u.system("• Built in Tic Tac Toe and Hangman! Run ./tic or ./hang <word> to start new games.")
+	u.system("• Emoji replacements! (like on Slack and Discord)")
```

Comment:
I don't like how there's so many calls to u.system, please replace this with broadcast() like in the original help message

Reply:
I do agree spamming system here kinda sucks, although broadcast also sucks. I tried embedding a file although that had a lot of issues (some newlines wouldnt register). I say we let this sit for now and solve this properly in another pr?

Reply:
Just use u.writeln and sub in the message being used rn in main

Reply:
wdym broadcast sucks